libecho/examples/RawConnection/main.cc

```diff
@@ -1,10 +1,14 @@
 //#include "Echo.h"
 #define _GLIBCXX_USE_C99 1
 #include <Echo.h>
+#include <chrono>
 #include <cstring>
+#include <fstream>
```

Comment:
Why is fstream needed?

Reply:
wasn't it in master? i'll check and remove if it's not needed